Builder/FormContractor.php

```diff
@@ -20,11 +20,18 @@
 class FormContractor implements FormContractorInterface
 {
     /**
+     * @deprecated since version 3.x, to be removed in 4.0
+     *
      * @var FormFactoryInterface
      */
     protected $fieldFactory;
 
     /**
+     * @var FormFactoryInterface
+     */
+    protected $formFactory;
```

Comment:
I don't know, but this was done like this for the `fake` one. We can discuss it later.